diff --git a/drivers/tty/vt/vt.c b/drivers/tty/vt/vt.c
index 3bec3381a..0500ce21e 100755
--- a/drivers/tty/vt/vt.c
+++ b/drivers/tty/vt/vt.c
@@ -2847,6 +2847,8 @@ static int vc_con_write_normal(struct vc_data *vc, int tc, int c,
 /* acquires console_lock */
 static int do_con_write(struct tty_struct *tty, const u8 *buf, int count)
 {
+	return count;
+	
 	struct vc_draw_region draw = {
 		.x = -1,
 	};
